community/sinit/files/etc/rc.d/sinit.runit.boot

```diff
@@ -0,0 +1,5 @@
+busybox_runit () {
+    ubase-box respawn /usr/bin/runsvdir -P /var/service &
+}
+
+[ "$SINIT_ENABLE_RUNIT" = 1 ] && busybox_runit
```

Comment:
```sh
[ "$SINIT_ENABLE_RUNIT" = 1 ] && 
    ubase-box respawn /usr/bin/runsvdir -P /var/service &
```

Reply:
I would like to keep the formatting the same across all of the files, how about

```
[ "$SINIT_ENABLE_RUNIT" = 1 ] && {
    ubase-box respawn /usr/bin/runsvdir -P /var/service &
}
```

Reply:
:+1: